tests/unit_tests.rs

```diff
@@ -2,7 +2,8 @@ use core::convert::TryFrom;
 
 use rand::thread_rng;
 
-use ed25519_zebra::{Signature, SigningKey, VerificationKey, VerificationKeyBytes};
+use ed25519::Signature;
+use ed25519_zebra::{SigningKey, VerificationKey, VerificationKeyBytes};
```

Comment:
With the re-export above we should be able to restore the original import

```suggestion
use ed25519_zebra::{Signature, SigningKey, VerificationKey, VerificationKeyBytes};
```